package.json

```diff
@@ -53,14 +53,14 @@
 	"devDependencies": {
 		"@namics/stylelint-bem": "^8.1.0",
 		"@rushstack/eslint-patch": "^1.2.0",
-		"@types/jest": "^29.5.0",
+		"@types/jest": "^29.5.1",
```

Comment:
This change doesn't belong here. The commit should only have the stylelint-config-wikimedia package